retriever/lib/engine_tools.py

```diff
@@ -76,12 +77,34 @@ def name_matches(scripts, arg):
     if arg == 'all':
         return scripts
 
+    from retriever.lib.scripts import get_script_version_upstream, get_script_upstream
```

Comment:
from retriever.lib.scripts import get_script_version_upstream, get_script_upstream
why do we have the import here?